chia/protocols/wallet_protocol.py

```diff
@@ -19,7 +20,7 @@
 """
 
 
-__all__ = ["CoinState", "RespondToPhUpdates"]
+__all__ = ["CoinState", "RespondToPhUpdates", "RespondFeeEstimates"]
```

Comment:
can you explain why this is needed ?
from what i understand its relevant to types that we implement in rust

Reply:
You can think of this as publicly exporting these types. I think I had to do this because my local version of mypy had lagged behind whatever the CI system was using.

Removed, thanks!